### ISSUE TITLE
Scale to full size of high res image.

### ISSUE THREAD

**Issue by SamesJeabrook** [kingdido999/zooming#317]
is there a way to set scaleExtra to scale to 100% of the images original size.

To fully understand the behaviour. I set a smaller size image, this is opened and the large size image (high res) is loaded as replacement and it moves to the modal. When I click to zoom in further I would like to zoom into the maximum size of the high res image. Is this currently possible?

**Comment by kingdido999** [comment]
No, this is currently not possible. `scaleExtra` is addition to the current scale of the opened image using [css transform scale function](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/scale).
